.drone.yml

```diff
@@ -1,46 +1,66 @@
 ---
+kind: pipeline
+type: docker
+name: default
 
-workspace:
-  base: /go
-  path: src/github.com/packethost/cacher
-
-pipeline:
-  lint:
-    group: ci
-    image: golangci/golangci-lint:v1.24.0
+steps:
+  - name: lint
+    image: golangci/golangci-lint:v1.23.8
     commands:
-      - golangci-lint run -v -D errcheck
+      - golangci-lint run -v --skip-dirs cmd/packetflow
```

Comment:
cmd/cacherc?

Reply:
fixed, I copied from hegel but failed to `git checkout -p` properly